daily_word_bot/app.py

```diff
@@ -25,7 +25,10 @@
     BotCommand("/help", "Opens help section"),
     BotCommand("/start", "Starts sending words"),
     BotCommand("/stop", "Stops sending words"),
-    BotCommand("/blockedwords", "Shows your blocked words")
+    BotCommand("/blockedwords", "Shows your blocked words"),
+    BotCommand("/mylevels", "Shows the level of the words you want to be sent: beginner, intermediate or advanced"),
+    BotCommand("/addlevel", "Adds to your levels a level of the words you wish to receive"),
+    BotCommand("/removelevel", "Removes from your levels a level of the words you wish to receive"),
```

Comment:
I got an enhancement for this 💡 

/mylevels commands shows in the reply_markup the list of current_levels stored. So that the user can toggle the levels just by clicking on them:

something like:
```
                                                                          -------------
                                                                         | /mylevels   |___
                                                                         ------------- ----
-----------------------------------------------
| Those are your levels:                      |
| - click ➕ for adding or ❌ to removing it |
-----------------------------------------------
| ❌beginner                                  |
-----------------------------------------------
| ❌intermediate                              |
-----------------------------------------------
| ➕advanced                                  |
-----------------------------------------------


```

In this way the user does not need to type the level value but just click on the button.
Including the an emoji would determine which levels are currently active for the user

Reply:
If done so commands **/addlevel** & **/removelevel** are then not needed which make the conversation flow more user friendly 😄 
 -  you would then include the callback's in `inline_keyboard_callbacks` method

Reply:
You are right, it would be much smoother and nicer. I though about the inlines but I though it would be too much effort for a first mvp version of the feature. But I really like the idea, I'll work on it hehe

Reply:
True, if you think it's too much effort I leave it up to you :)

Reply:
I think I solved everything mentioned in the review and I even added the enhancement you proposed. Feel free to check it out and give me some feedback :)